packages/core/bootstrap/src/lib/ws/epics.ts

```diff
@@ -302,12 +347,90 @@ export const connectEpic: Epic<AnyAction, AnyAction, { ws: RootState }, any> = (
         }),
       )
 
+      const withOnConnectChainComplete$ = action$.pipe(
+        filter((action) => onConnectComplete.match(action) && !!wsHandler.heartbeatMessage),
+        withLatestFrom(state$),
+        filter(([action, state]) => {
+          const connectionKey = action.payload.connectionInfo.key
+          const connectionState = state.ws.connections.all[connectionKey]
+          return !!connectionState && connectionState.active
+        }),
+        mergeMap(([action, state]) => {
+          const connectionKey = action.payload.connectionInfo.key
+          const connectionState = state.ws.connections.all[connectionKey]
+          const interval = wsHandler.heartbeatIntervalInMS || 30000 // Default Interval
```

Comment:
```suggestion
          const interval = wsHandler.heartbeatIntervalInMS || DEFAULT_HEARTBEAT_INTERVAL
```
Let's pull this out. Might also want to make it configurable via env var.